alpha/lib/index/BaseIndexObject.php

```diff
@@ -228,4 +228,31 @@ public static function getCacheInvalidationKeys($object = null)
 	{
 		return array();
 	}
+
+	public static function getSphinxSplitIndexFieldName()
+	{
+		return null;
+	}
+	
+	public static function getSplitIndexFactor($IndexObjectName)
+	{
+		$config = kConf::getDB();
+		if(!isset($config['sphinx_split_index']) || !$config['sphinx_split_index']['enabled'] || $config['sphinx_split_index']['enabled'] == false || !isset($config['sphinx_split_index'][$IndexObjectName]))
+			return null;
+		
+		return $config['sphinx_split_index'][$IndexObjectName];
+		
+	}
+	
+	public static function getSphinxSplitIndexId($originalValue = null, $IndexObjectName)

```

Comment:
Is there a case where $originalValue can be null? or will be called without values? If so why originalValue  with default value  is not last